src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -71,7 +71,9 @@ def _print_detailed_help(self, cli_name, help_file):
     def _get_choices_defaults_sources_str(p):
         choice_str = u'  Allowed values: {}.'.format(', '.join(sorted([str(x) for x in p.choices]))) \
             if p.choices else ''
-        default_str = u'  Default: {}.'.format(p.default) if p.default and p.default != argparse.SUPPRESS else ''
+        default_value_source = p.default_value_source if p.default_value_source else u'Default'
```

Comment:
`u''` notation is deprecated in Python 3: https://stackoverflow.com/a/2464968/2199657

You're right, see *[3.1.3. Unicode Strings][1]*.

It's been the syntax since Python 2.0. 

Python 3 made them redundant, as the default string type is Unicode. Versions 3.0 through 3.2 removed them, but they were [re-added in 3.3+][pep414] for compatibility with Python 2 to aide the 2 to 3 transition.

  [1]: http://docs.python.org/2/tutorial/introduction.html#unicode-strings
  [pep414]: https://www.python.org/dev/peps/pep-0414/

Reply:
all `u` in `_help.py` were removed now.